From 72ae3128b3d515e776729e6cd714b8af34280901 Mon Sep 17 00:00:00 2001
From: Amarnath Valluri <amarnath.valluri@intel.com>
Date: Fri, 7 Jul 2017 12:18:15 +0300
Subject: [PATCH 2/4] Fix rpcd plugins install location

Signed-off-by: Amarnath Valluri <amarnath.valluri@intel.com>
---
 luci2/src/rpcd/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/luci2/src/rpcd/CMakeLists.txt b/luci2/src/rpcd/CMakeLists.txt
index 3621995..8b763f4 100644
--- a/luci2/src/rpcd/CMakeLists.txt
+++ b/luci2/src/rpcd/CMakeLists.txt
@@ -23,4 +23,4 @@ SET_TARGET_PROPERTIES(luci2-plugin PROPERTIES OUTPUT_NAME luci2 PREFIX "")
 ADD_LIBRARY(bwmon-plugin MODULE bwmon.c)
 SET_TARGET_PROPERTIES(bwmon-plugin PROPERTIES OUTPUT_NAME bwmon PREFIX "")
 
-INSTALL(TARGETS luci2-plugin bwmon-plugin LIBRARY DESTINATION lib)
+INSTALL(TARGETS luci2-plugin bwmon-plugin LIBRARY DESTINATION lib/rpcd)
-- 
2.7.4

