From 97e501de1ce58e8d924473e9d76cd77841c7897e Mon Sep 17 00:00:00 2001
From: Amarnath Valluri <amarnath.valluri@intel.com>
Date: Wed, 2 Aug 2017 14:36:55 +0300
Subject: [PATCH 2/2] session: Handle disabled accounts properly

Signed-off-by: Amarnath Valluri <amarnath.valluri@intel.com>
---
 session.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/session.c b/session.c
index 951201b..30c7ff7 100644
--- a/session.c
+++ b/session.c
@@ -800,6 +800,11 @@ rpc_login_test_password(const char *hash, const char *password)
 		return true;
 	}
 
+    /* disabled account */
+    if (!strcmp(hash, "*")) {
+        return false;
+    }
+
 	/* password hash refers to shadow/passwd */
 	else if (!strncmp(hash, "$p$", 3))
 	{
-- 
2.7.4

