From 1e016bfab513c284a73300478cc366a7a7f0185a Mon Sep 17 00:00:00 2001
From: Amarnath Valluri <amarnath.valluri@intel.com>
Date: Tue, 22 Aug 2017 16:21:56 +0300
Subject: [PATCH 2/4] fix build failure when builddir is different from source
 dir

Signed-off-by: Amarnath Valluri <amarnath.valluri@intel.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 5861b6e..0f287e7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,7 +4,7 @@ SUBDIRS=src test
 EXTRA_DIST=autogen.mk
 
 node_modules: node_modules.tar.gz 
-	@if [ ! -d node_modules ]; then tar -xzf node_modules.tar.gz; fi
+	@if [ ! -d node_modules ]; then tar -xzf ${top_srcdir}/node_modules.tar.gz; fi
 
 all: node_modules
 
-- 
2.7.4

